modules/root/main.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  context = {
+    # Used by `accounts`
+    accounts_enabled = "${jsonencode(var.accounts_enabled)}"
+    account_email    = "${var.account_email}"
+
+    # Used by `root-dns`
+    root_domain_name = "${var.stage}.${var.domain}"
+  }
+
+  vars = "${merge(var.vars, local.context)}"
```

Comment:
Here https://github.com/cloudposse/reference-architectures/pull/1/files#diff-27326578547e57e618f597f0e195305aR29

we have 
```
vars = "${merge(local.context, var.vars)}"
```

meaning `var.vars` could override local context.

Make the same for consistency?

Reply:
Reversed the other one to work like this one.